src/drive/web/modules/filelist/duck.js

```diff
@@ -28,3 +30,6 @@ export default filelist
 
 export const isTypingNewFolderName = state =>
   state.filelist.isTypingNewFolderName
+
+export const isThereFileWithThisMetadata = (files, attribute) =>
+  files.some(file => get(file, `metadata.${attribute}`))
```

Comment:
If we change the logic and abstract the metadata attribute getting, I think our code will be more expressive, and more reusable.

```
export const hasMetadataAttribute = (file, attribute) => get(file, `metadata.${attribute}`)

// And then in the caller, it is more understandable and we do not have
// to reinvent the some API
files.some(x => hasMetadataAttribute(x, 'carbonCopy'))
```